2.4/Dockerfile.fedora

```diff
@@ -0,0 +1,79 @@
+# FROM registry.fedoraproject.org/f26/s2i-base:latest
+FROM f27/s2i-core
+
+# This image provides a Ruby 2.4 environment you can use to run your Ruby
+# applications.
+
+EXPOSE 8080
+
+ENV NAME=ruby \
+    RUBY_VERSION=2.4 \
+    VERSION=0 \
+    RELEASE=1 \
+    ARCH=x86_64
+
+ENV SUMMARY="Platform for building and running Ruby $RUBY_VERSION applications" \
+    DESCRIPTION="Ruby $RUBY_VERSION available as docker container is a base platform for \
+building and running various Ruby $RUBY_VERSION applications and frameworks. \
+Ruby is the interpreted scripting language for quick and easy object-oriented programming. \
+It has many features to process text files and to do system management tasks (as in Perl). \
+It is simple, straight-forward, and extensible."
+
+LABEL summary="$SUMMARY" \
+      name="$FGC/$NAME" \
+      com.redhat.component="$NAME" \
+      release="RELEASE.$DISTTAG" \
+      architecture="$ARCH" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Ruby 2.4" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,ruby,ruby24,rh-ruby24" \
+      usage="s2i build <SOURCE-REPOSITORY> $FGC/ruby <APP-NAME>" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+
+# Install required packages
+RUN INSTALL_PKGS="bsdtar \
+    findutils \
+    gettext \
+    tar \
+    bzip2 \
+    gcc-c++ \
+    gd-devel \
+    gdb \
+    git \
+    libcurl-devel \
+    libxml2-devel \
+    libxslt-devel \
+    lsof \
+    make \
+    community-mysql-devel \
+    npm \
+    openssl-devel \
+    patch \
+    postgresql-devel \
```

Comment:
Is this really needed for rh-ruby24?

Reply:
I'm assuming that all the `-devel` packages are installed so that you can compile gems which need the header files. It's really up to you whether you want to leave it, I can easily remove.

Reply:
Well I can see in general why that's useful, although I do not know how the list was compiled. That would be perhaps good to know.

I was specfically pointing to `postgresql-devel`, but now, come to think of it, it would seem that people would want use postgresql anyway.